unsplahAPI/unsplahAPI/Authentification/AuthentificationCoordinatorProtocol.swift

```diff
@@ -4,8 +4,10 @@
 //
 //  Created by Никита Данилович on 25.05.2023.
 //
-
+import UIKit
```

Comment:
A bit of spacing between imports and protocols 😄 
```suggestion
import UIKit

```

Reply:
thank you, will be fixed!